X/XicTools/build_tarballs.jl

```diff
@@ -0,0 +1,130 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "XicTools"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/wrcad/xictools", "c7a50a5fcd71966730e45a5358b8507227ae098c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/xictools/
+cp Makefile.sample Makefile
+sed -i "s|enable-itopok=no|enable-itopok=yes|g" Makefile
+sed -i "s|GFXLOC = --enable-gtk2=yes|#GFXLOC = --enable-gtk2=yes|g" Makefile
+#sed -i "s|^SUBDIRS =.*|SUBDIRS = xt_base adms KLU vl wrspice|g" Makefile
+sed -i "766c\#include <malloc.h>" xt_base/malloc/malloc-2.8.6.c
+update_configure_scripts
+make config
+make all -j$(nproc)
+make prefix="${prefix}/usr" install
+install_license ${WORKSPACE}/srcdir/xictools/license/LICENSE-2.0.txt
+cd ${prefix}/usr/xictools
+install -Dvm 755 wrspice.current/bin/mmjco "${bindir}/mmjco"
```

Comment:
This can probably be simplified with something like
```bash
for file in wrspice.current/bin/mmjco wrspice.current/bin/multidec ...; do
    install -Dvm 755 "${file}" "${bindir}/$(basename ${file})"
    rm "${file}"
done
```

Reply:
Fixed with something similar to above. Also added symlinks to deleted files because I noticed BinaryBuilder warnings (and more importantly may cause breakage in the program).